.github/CONTRIBUTING.md

```diff
@@ -92,13 +94,13 @@ If you want to check if your code complies to the style guide, you can run:
 
 Imports must be sorted in the following order
 
+1. `import static java.*`
+1. `import static javax.*`
+1. `import static all other imports`
+1. blank line
 1. `import java.*`
-2. `import javax.*`
-3. `import all other imports`
-4. blank line
-5. `import static java.*`
-6. `import static javax.*`
-7. `import static all other imports`
+1. `import javax.*`
+1. `import all other imports`
```

Comment:
I think we should also leverage [IntelliJ IDEA's `.editorconfig` extensions](https://www.jetbrains.com/help/idea/configuring-code-style.html#editorconfig) for that.